core/dataset.cpp

```diff
@@ -801,36 +858,12 @@ Dataset operator+(const Dataset &lhs, const DataConstProxy &rhs) {
   return res;
```

Comment:
`apply_with_broadcast`?

Reply:
What would you expect to happen here? Would the `DataConstProxy` operate on every data item in the `Dataset`?

Reply:
Yes, same as now.